src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/BalanceSheetControllerTests.java

```diff
@@ -68,6 +68,8 @@
 
     private static final String BACK_BUTTON_MODEL_ATTR = "backButton";
 
+    private static final String TEMPLATE_NAME_MODEL_ATTR = "templateName";
```

Comment:
Can you assert this attribute exists when the page is redisplayed after an error?